features/step_definitions/networking.rb

```diff
@@ -1275,3 +1276,10 @@
   cb[cb_name]="172.31.249.80:9095"
   logger.info "The ipecho service url #{cb[cb_name]} is stored to the #{cb_name} clipboard."
 end
+
+Given /^the IPSec is enabled on the cluster$/ do
+  ensure_admin_tagged
+  _admin = admin
+  @result = _admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.defaultNetwork.ovnKubernetesConfig}")
```

Comment:
Yeah we can do that but we need to dig until ```ovnKubernetesConfig``` so it would be like
```rr.dig('spec', 'defaultNetwork', 'ovnKubernetesConfig')```

```
spec:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  defaultNetwork:
    ovnKubernetesConfig:
      genevePort: 6081
      ipsecConfig: {}
      mtu: 8855
    type: OVNKubernetes
```